app/waf/src/controllers/application.controller.ts

```diff
@@ -264,19 +263,16 @@ export class ApplicationController {
           },
```

Comment:
The `params.endpointpolicy` may not exist.

Reply:
corrected it

Reply:
corrected it